v20.1/enable-node-map.md

```diff
@@ -137,46 +140,46 @@ For the latitudes and longitudes of AWS, Azure, and Google Cloud regions, see [L
 
 The Localities debug page displays the following:
 
 - Localities configuration that you set up while starting the nodes with the `--locality` flags.
 - Nodes corresponding to each locality.
 - Latitude and longitude coordinates for each locality/node.
 
-On the page, ensure that every node has a locality as well as latitude/longitude coordinates assigned to them.
+On the page, ensure that each node is assigned a locality and latitude/longitude coordinates.
 
 ### Node Map not displayed for all locality levels
 
 The **Node Map** is displayed only for the locality levels that have latitude/longitude coordinates assigned to them:
 
-- If you assign the latitude/longitude coordinates at the region level, the **Node Map** shows the regions on the world map. However, when you drill down to the datacenter and further to the individual nodes, the world map disappears and the datacenters/nodes are plotted in a circular layout.  
-- If you assign the latitude/longitude coordinates at the datacenter level, the **Node Map** shows the regions with single datacenters at the same location assigned to the datacenter, while regions with multiple datacenters are shown at the center of the datacenter coordinates in the region. When you drill down to the datacenter levels, the **Node Map** shows the datacenter at their assigned coordinates. Further drilling down to individual nodes shows the nodes in a circular layout.
+- If you assign the latitude/longitude coordinates at the region level, the Node Map shows the regions on the world map. However, when you drill down to the datacenter and further to the individual nodes, the world map disappears and the datacenters/nodes are plotted in a circular layout.  
+- If you assign the latitude/longitude coordinates at the datacenter level, the Node Map shows the regions with single datacenters at the same location assigned to the datacenter, while regions with multiple datacenters are shown at the center of the datacenter coordinates in the region. When you drill down to the datacenter levels, the Node Map shows the datacenter at their assigned coordinates. Further drilling down to individual nodes shows the nodes in a circular layout.
 
-[Assign latitude/longitude coordinates](#step-3-set-the-latitudes-and-longitudes-for-the-localities) at the locality level that you want to view on the **Node Map**.
+[Assign latitude/longitude coordinates](#step-3-set-the-latitudes-and-longitudes-for-the-localities) at the locality level that you want to view on the Node Map.
 
 ## Known limitations
 
 ### Unable to assign latitude/longitude coordinates to localities
 
 {% include {{ page.version.version }}/known-limitations/node-map.md %}
 
-### **Capacity Used** value displayed is more than configured Capacity
+### Displayed **Used Capacity** value exceeds configured Capacity
```

Comment:
I think "is more than" is much easier to search for than "exceeds"

Maybe something like 

"Used Capacity is more than configured Capacity" ?